Process_Library/GOCART2G_Process.F90

```diff
@@ -2392,23 +3549,28 @@ subroutine Aero_Compute_Diags (mie_table, km, klid, nbegin, nbins, rlow, rup, ch
     end if !present(extcoef)...
 
    if( (present(exttau) .and. associated(exttau)) .or. &
```

Comment:
@rmontuoro I talked with @tclune and he agrees this is an issue. For now we will approve (since Intel is at least "working") and merge what is here but I'll make an issue to fix this. I don't want to try and untangle this:

https://github.com/GEOS-ESM/GOCART/blob/58af7747eba91dc597c65485a5e4e781cb25deca/Process_Library/GOCART2G_Process.F90#L3551-L3558

on a Friday afternoon!

Reply:
Issue #72 created

Reply:
Actually, this is not quite as bad as I first thought.   Can be converted to 2 conditionals:
```f90
if (present(exttau) .and. present(stexttau) .and. ...) then
   if (associated(exttau) .and. associated(stexttau) .and. ...) then
       ...

```